pkg/variables/variables.go

```diff
@@ -66,8 +61,13 @@ func (vars *Variables) Map() map[string]interface{} {
 	return m
 }
 
-// Merge merges two Containers into new one
-func (vars *Variables) Merge(src Container) Container {
+// Merge merges into current container with the src Container
+// src will overwrite the existing keys if exists
+// returns a the merged Container
+//
+// TODO: since we are dealing with pointers this does not need to return anything
+// instead update only the underlying reference, create new story for this.
```

Comment:
Is this a TODO for this PR?

Reply:
next PR - ideally no more changes in this one 🤔